hail/python/hailtop/google_storage.py

```diff
@@ -112,6 +120,13 @@ async def wrapped(*args, **kwargs):
                                            **kwargs)
         return wrapped
 
+    def _get_etag(self, uri: str):
+        b = self._get_blob(uri)
+        if b.exists():
+            b.reload()
+            return b.etag
+        return None
```

Comment:
I fixed 2. Going to skip 1. since I didn't introduce `get_blob` in this PR and I'm using the non-wrapped/async version here, anyway and `get_etag` should be wrapped since it does make an API call.